cloud/datasync/example/index.ts

```diff
@@ -1,6 +1,8 @@
+import { LoggerManager } from '@raincatcher/logger';
 import SyncServer, { SyncApi, SyncExpressMiddleWare, SyncOptions } from '../src/index';
 
 const sync: SyncApi = SyncServer;
+const log = new LoggerManager();
```

Comment:
Is that using empty logger? I'm missing part when we setup some particular logger.